source/includes/entityref/paybill/_billmaster.md

```diff
@@ -1,7 +1,10 @@
 # Pay and Bill - BillMaster
 
-Read-only entity that represents a BillMaster record.
+Represents a Bill Master record.
```

Comment:
Change "Bill Master" back to "BillMaster" for consistency with doc for "PayMaster". Tool automatically splits it but we want it CamelCased here.